docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3376,6 +3376,25 @@ The same can be achieved by adding this rule:
 
 We recommend to use this kind of exceptions only if it is not possible to change the hiding rule itself. In other cases it is better to change the original rule, using domain restrictions.
 
+:::note CSS injection
```

Comment:
I think we should move it to CSS rules section and call it "Compatibility with Adblock Plus".